docs/en/data/people.yml

```diff
@@ -0,0 +1,546 @@
+maintainers:
+- login: tiangolo
+  answers: 1844
+  prs: 430
+  avatarUrl: https://avatars.githubusercontent.com/u/1326112?u=740f11212a731f56798f558ceddb0bd07642afa7&v=4
+  url: https://github.com/tiangolo
+experts:
+- login: Kludex
+  count: 434
+  avatarUrl: https://avatars.githubusercontent.com/u/7353520?u=62adc405ef418f4b6c8caa93d3eb8ab107bc4927&v=4
+  url: https://github.com/Kludex
+- login: dmontagu
+  count: 237
+  avatarUrl: https://avatars.githubusercontent.com/u/35119617?u=540f30c937a6450812628b9592a1dfe91bbe148e&v=4
+  url: https://github.com/dmontagu
+- login: Mause
+  count: 220
+  avatarUrl: https://avatars.githubusercontent.com/u/1405026?v=4
+  url: https://github.com/Mause
+- login: ycd
```

Comment:
can you get rid of these people? they're FastAPI's people not us haha
Maybe you can add us as maintainers :)